### ISSUE TITLE
German Translation

### ISSUE THREAD

**Issue by davidcarlisle** [learnlatex/learnlatex.github.io#71]
This is a placeholder to mark the German translation started with PR #64 by @sieversMartin 

See https://www.learnlatex.org/TRANSLATIONS#initial-setup

just so that the [translations tag](https://github.com/learnlatex/learnlatex.github.io/labels/translation) lists currently active translations.

**Comment by davidcarlisle** [comment]
Thank you  for working on learnlatex.org translations.
You may have seen already if watching the repository but just a note
to say that we have extended lesson-06 and more-06 to add descriptions
of defining commands with \newcommand and \NewDocumentCommand

https://github.com/learnlatex/learnlatex.github.io/commit/463d181301d667b10bf23c164f9d5ae15371877e

**Comment by sieversMartin** [comment]
Thanks for the heads-up. I should really watch the project more closely ;-)

I will work on the translations and send you a pull-request.

**Comment by Skillmon** [comment]
@sieversMartin You don't have to do it all by yourself, I can help as well. Do you have any sections you'd prefer to do or sections you'd rather not translate?

**Comment by sieversMartin** [comment]
Well I started with `more-06.md` this morning and would like to finish it. If you would like to start with `lessions-06` please go ahead and let me know, what's left.

**Comment by davidcarlisle** [comment]
@sieversMartin  do you have translations of other lessons that you have not pushed? Currently we only have the home page in German and none of the lessons?

**Comment by sieversMartin** [comment]
@davidcarlisle That is correct. I haven't had time to do more back then. However, I'd like to contribute what's needed. Do you want me to start somewhere specific?

**Comment by davidcarlisle** [comment]
That is fine, everyone is volunteering do not feel pressured to use time that you don't have

it is just that the wording you used to @Skillmon suggested that if you did more-06  and he did lesson-06 there would be a question as to "what is left" whereas it would seem to be every other lesson, so I just wanted to check that we were not missing an update.

> Do you want me to start somewhere specific?

Currently the partial translations have been removed from the menu on the public site, so the pages can be worked on in any order, once they are all there we can re-enable German in the language switch menu to make them easily available.


David

**Comment by sieversMartin** [comment]
To be honest, I forgot, that the lessons are still to be done and therefore start with the modified `more-06`.

**Comment by Skillmon** [comment]
@sieversMartin Where do you push to? Do you have some fork set up that I can get writing access to so that we can work on one branch, or should we keep a list of things we've already done in somewhere and work entirely separated?

**Comment by sieversMartin** [comment]
@Skillmon  Feel free to push to my fork. I sent you an invitation.

**Comment by sieversMartin** [comment]
@Skillmon Do you prefer "Sie" or "Du" when speaking to the user.

**Comment by jejust** [comment]
@sieversMartin In the French version, I stayed a bit formal, using « vous », and not « tu » (meaning we're considering the reader is a grown up, not a kid), FYI.

**Comment by Skillmon** [comment]
@sieversMartin That's a good question I haven't decided upon yet, I translated lesson-01 and more-01 (haven't pushed yet) in which I rephrased such that I don't directly address the user. I think I'd prefer the formal version (like @jejust did in the French translation), but maybe we should also ask @davidcarlisle and @josephwright.

**Comment by davidcarlisle** [comment]
I think it would be best if you didn't rephrase not to directly address the user (where we have used "you" in English") one can argue about the style but that style has been used fairly consistently  in the English version and as a general policy keeping the translations as close as possible would be good.

As to which word to use, well I leave that to you:-)

**Comment by Skillmon** [comment]
In this case I'd prefer the formal approach, but comparing to e.g. learn-c.org, they use the informal "du".

**Comment by marmotghost** [comment]
@davidcarlisle "You" has an official German translation to "man", which solves the problem. It isn't really impolite, but "Du" or "Sie" can be depending who reads it. There are also huge regional differences.

**Comment by davidcarlisle** [comment]
@marmotghost  really the reason why we ask real humans to translate this not just pipe it through google translate is to get these things right and not rely on machines or Englishmen. So doing whatever seems necessary is fine. As a general rule it would be good to keep the references to the reader in more or less the same style as the original rather than rephrasing to avoid it, but choice of pronoun is really something I can't have any opinion on. If Grammar or political niceties make that awkward, then rephrasing as @Skillmon said he had tried in lesson-01 is a possibility.

There are around 300 "you" in the en/*.md pages though, so a form that minimises change from the literal translation would have some advantages....

**Comment by marmotghost** [comment]
@davidcarlisle OK, @Skillmon and I will try to find a human. Obviously neither rabbits nor marmots qualify. ;-)
A bit more seriously, you can compare this situation to textbooks or lecture notes. While you will find many "you"s in English texts, you will have to search very long till you find a "Sie" or "Du" in German books or notes. For instance, https://web.ma.utexas.edu/users/a.debray/lecture_notes/m392c_QFT_notes.pdf has over 200 "you"s, not just in exercises, and I'd believe that in a German translation you may find a handful of "Du"s or "Sie"s (but nowadays such things do not really get translated much anyway ;-)).

**Comment by davidcarlisle** [comment]
As you may have seen, the main site at learnlatex.org has been given a new design and some of the details of the language data have changed. 

If you are still working on the translation, I have updated
https://www.learnlatex.org/TRANSLATIONS
to hopefully give correct instructions for the new layout, but if anything is not clear please ask
